torch_xla/csrc/aten_cuda_functions.cpp

```diff
@@ -11,6 +11,8 @@ static void fail(const char* name) {
 
 namespace c10::cuda {
 
+DeviceIndex device_count() noexcept { return 0; }
```

Comment:
It returns 0 because it's a phony implementation? 
Also, could you add a comment in this file describing it's phony implementation and what is the purpose of this file?